ports/net/macathena-libgms/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
```

Comment:
Why do you need libgms? get_message is pure Python now, I don't think it depends on libgms at all (which is pretty much dead at this point).